### PR TITLE
change tiles types

### PR DESCRIPTION
**Pull request by light5551**
https://docs.google.com/document/d/1R8MHrdkn7lliGWpqRMOH4n3_kP0FwMjlvEUbnJCFuP4/edit

### CODE REVIEW

File: src/dt_maps/utils/tiles.py [hunk 1]
```diff
@@ -1,11 +1,28 @@
...
     return None
+
+
+def orientation2deg(tile_orientation: TileOrientation) -> int:
+    return TILE_ORIENTATIONS[tile_orientation]
+
+
+def deg2orientation(tile_deg: int) -> Optional[TileOrientation]:
+    for obj_orientation, deg in TILE_ORIENTATIONS.items():
+        if tile_deg == deg:
```
Comment (afdaniele) [inline src/dt_maps/utils/tiles.py]: Let's use a soft-match here, an exact math will likely fail.

File: tests/maps/loop/tiles.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ tiles:
     map_0/tile_0_0:
         i: 0
         j: 0
-        type: curve_left
+        type: curve
         orientation: N
```
Comment (afdaniele) [inline tests/maps/loop/tiles.yaml]: `orientation` should not be a field anymore, the orientation is only defined in the `frames` layer.

**Review by afdaniele — CHANGES_REQUESTED** [review]

File: tests/maps/loop/tiles.yaml (hunk 2 above)
Comment (light5551) [reply tests/maps/loop/tiles.yaml]: Ok, so i removed orientation in tile layer

File: src/dt_maps/utils/tiles.py (hunk 1 above)
Comment (light5551) [reply src/dt_maps/utils/tiles.py]: https://github.com/duckietown/lib-dt-maps/pull/5#discussion_r896623663 - remove unused functions

**Comment by light5551** [comment]
I have updated the tests, because now we have only curve instead of curve_left, curve_right and remove orientations

**Review by afdaniele — APPROVED** [review]

**Pull request merged**